internal/lib/lib.go

```diff
@@ -1460,3 +1460,8 @@ func IsVCFCluster() bool {
 
 var VCFInitialized bool
 var AviSecretInitialized bool
+
+func GetRetainSharedVSIPs() bool {
+	retainSharedVSIPs := os.Getenv(RETAIN_SHARED_VS_IPS)
```

Comment:
it would be good to do a `parseBool` check here instead of doing a string compare.

Reply:
I have removed the string comparison and added the parseBool check as per the comment.